index.html

```diff
@@ -0,0 +1,496 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component Library</title>
+    <script src="https://kit.fontawesome.com/368288da44.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+   
+</head>
+<body>
+    <!-- avatar-->
+    <div class="information">
+        <header>Avatar</header>
+        <p>Avatar can be used to show users profile picture.</p>
+        <h3>Example of image Avatar</h2>
+    </div>
+    <div class="avatar">
+        <img class="img1" src="https://www.w3schools.com/w3images/avatar2.png" alt="image">
+        <img class="img2" src="https://www.w3schools.com/w3images/avatar2.png" alt="image">
+        <img class="img3" src="https://www.w3schools.com/w3images/avatar2.png" alt="image">
+    </div>
+    <!-- alert-->
+        <div class="information">
+            <header>Alert</header>
+            <p>Badges are being used to display a notification count or status information</p>
+            <h3>Example of different Alerts</h2>
```

Comment:
Suggestion : Try not to use colors in heading. I was also doing this, but since its a heading, you can skip adding colors to those